### DISCUSSION TITLE
暗色模式的滚动条是亮色模式的

### DISCUSSION THREAD

**Discussion by Envov** [geeknote-net/feedback#34] [General]
<img width="1536" alt="image" src="https://user-images.githubusercontent.com/48908684/227828424-deabea02-7882-44c7-8e61-204a7910d717.png">
参考github的暗色模式
<img width="1534" alt="image" src="https://user-images.githubusercontent.com/48908684/227828453-495c8e1c-56dd-4587-b96b-df138530c5ae.png">

**Comment by chloerei** [comment]
这是什么系统和浏览器，刷新之后还出现吗？我在 Mac 无法复现滚动条的亮色背景。

<img width="292" alt="截屏2023-03-27 16 39 14" src="https://user-images.githubusercontent.com/49931/227889092-f382ae40-7c1c-4d1c-9593-8208422f47d0.png">

**Reply by Envov** [reply]
是macos的chrome系统13.0.1
<img width="1632" alt="image" src="https://user-images.githubusercontent.com/48908684/228533128-f7b05660-fcfa-47e5-bf59-1dc9e9aa6832.png">

**Reply by Envov** [reply]
<img width="1667" alt="image" src="https://user-images.githubusercontent.com/48908684/228533499-ce5656d3-873f-4958-91f9-f3e5b8c084ac.png">
使用这个样式可以解决

**Reply by chloerei** [reply]
我加上了

```
@media (prefers-color-scheme: dark) {
body {
    color-scheme: dark;
    ...
```

可能是版本差异，我本地复现不了问题，不知道解决了没，麻烦反馈一下。

我系统版本是 macOS 13.2.1，Chrome 111.0.5563.110 。

**Comment by Envov** [comment]
没有解决
```
@media (prefers-color-scheme: dark) {
html {color-scheme: dark;}
```
这样可以解决，因为GeekNote的的最外层滚动条是在html上的，不是在body上，所以body的样式影响不了滚动条

**Reply by chloerei** [reply]
修改了，把样式上移到 html，有空再帮测试一下。

**Reply by Envov** [reply]
已经解决，感谢

**Comment by Envov** [comment]
顺便问一下大佬，GeekNote的写作体验很好，我想把它作为写作的地方，
非常需要api的功能，作为我个人博客的内容管理的地方(类似某些CMS)，我会使用一些ssg框架在构建阶段获取我的内容，然后发布到我的网站(同时也发布到Geeknote)，但是我不想做一个后台管理，也不想在仓库里面写文章
效果为：
我在Geeknote写作，通过一个按钮可以触发github的ci，我的博客会自动通过api静态生成内容
可以接受¥100/年以内的价格，如果付费的话，需要支持图片外链，和文章不可见(但是api可见)

**Reply by Envov** [reply]
如果作为免费内容，是不是可以接入文章评论系统(通过oauth)
用户发表评论需要到GeekNote注册一个账户(可以带来流量)，跳转回来后就可以评论
如果使用这种方式，GeekNote也可以收集到更多的内容，为我这样开发者也提供便捷，我觉得是一个非常好的模式

**Reply by chloerei** [reply]
很高兴你喜欢 GeekNote 的写作体验。

往外发布的需求我想了想，跟网站的目标不太符合。GeekNote 目标是想做一个技术社区，聚合文章和评论。倒是做了 RSS 导入，可以先发布到自己的网站，然后自动同步然后手工发布到 GeekNote。另外有 RSS Feed，可以用作程序抓取。图片外链容易被滥用，不敢开放。

暂时不会实现这个需求，但是保持开放态度。
